### PR TITLE
Generate test officers for development environment

### LINKED ISSUE #102: Make automated test data generation less ...manual
Should be an iterative thing, not declarative, from a hash or something containing a bunch of officers and subsequent data.

I'll grab it in the forthcoming days if no one else wants to grab it beforehand.

### PR DESCRIPTION
**Pull request by redshiftzero**
This PR adds automatic generation of test data: 120 officers are inserted into the development database with a randomly selected Springfield Police Department officer image. Closes #102. 

screen shot 2016-11-09 at 3 38 18 pm

### CODE REVIEW

**Comment by redshiftzero** [comment]
Tagging @r4v5 on this one to see if this is what you had in mind

**Comment by freddymartinez9** [comment]
based on the commits it LGTM

File: OpenOversight/test_data.py [hunk 1]
```diff
@@ -3,46 +3,48 @@
 import argparse
 from datetime import datetime
 import sys
+import random
```
Comment (r4v5) [inline OpenOversight/test_data.py]: Is python's random seedable?

File: OpenOversight/test_data.py [hunk 2]
```diff
@@ -52,15 +54,33 @@ def populate():
...
+
+        test_assignment = models.Assignment(star_no=pick_star(),
+                                            rank=pick_rank(),
+                                            officer=test_officer)
+
+        db.session.add(test_assignment)
+        db.session.commit()
+
+        test_face = models.Face(officer_id=test_officer.id,
+                                img_id=random.choice(test_images).id)
```
Comment (r4v5) [inline OpenOversight/test_data.py]: can we have some un-pictured officers here too?

Comment (redshiftzero) [inline OpenOversight/test_data.py]: Done!

**Comment by redshiftzero** [comment]
Thanks for your review and comments @r4v5! Merging...

**Pull request merged**